Anything I need to do to enable it? Still getting errors
> D:\home\site\repository\angularmvc\app\components\todolist\todolist.component.ts(1,25): error TS2307: Build:Cannot find module '@angular/core'. [D:\home\site\repository\angularmvc\angularmvc.csproj]

And many more

--- a/angularmvc/angularmvc.csproj
+++ b/angularmvc/angularmvc.csproj
@@ -26,7 +26,7 @@
     <UseGlobalApplicationHostFile />
     <NuGetPackageImportStamp>
     </NuGetPackageImportStamp>
-    <TypeScriptToolsVersion>2.0</TypeScriptToolsVersion>
+    <TypeScriptToolsVersion>2.1</TypeScriptToolsVersion>
     <SccProjectName>SAK</SccProjectName>
     <SccLocalPath>SAK</SccLocalPath>
     <SccAuxPath>SAK</SccAuxPath>

